.postman/collection.json

```diff
@@ -1517,6 +2063,111 @@
 							}
 						}
 					]
+				},
+				{
+					"name": "Skills",
+					"item": [
+						{
+							"name": "Get all skills",
+							"request": {
+								"method": "GET",
+								"header": [],
+								"url": {
+									"raw": "{{host}}/api/skills",
+									"host": [
+										"{{host}}"
+									],
+									"path": [
+										"api",
+										"skills"
+									]
+								}
+							},
+							"response": []
+						},
+						{
+							"name": "Create skill",
```

Comment:
What about adding this script as a test?
```js
skill_id = pm.response.json().data.id;
pm.environment.set("skill_id", skill_id);
```
That way, the `skill_id` environment variable would be fed to the endpoints that need it ("Update Skill" and "Delete skill"). You can take a look at how I made it on #107.